actions/flakeguard-ai-analysis/main.py

```diff
@@ -12,7 +12,7 @@
 
 
 def main():
-    file_path = Path(__file__).parent
+    file_path = Path("/github/workspace")
```

Comment:
Should this be a configurable input?

Also `$GITHUB_WORKSPACE` is an env that is also available, not sure if it's the directory you need though.

Slightly more information here: https://github.com/actions/checkout/issues/785

Reply:
so because this is running within a docker container, the mount path is always the same.

example:
```/usr/bin/docker run ... -v "/home/runner/work/chainlink/chainlink":"/github/workspace" ...```

from https://github.com/smartcontractkit/chainlink/actions/runs/14087667410/job/39461771982